src/main/java/com/revature/controllers/TripController.java

```diff
@@ -107,13 +129,74 @@
      */
     @ApiOperation(value = "Updates a trip by ID", tags = {"Trip"})
     @PutMapping
-    public ResponseEntity<Trip> updateTrip(@Valid @RequestBody Trip trip) {
+    public ResponseEntity<TripDTO> updateTrip(@Valid @RequestBody Trip trip) {
         Optional<Trip> existingTrip = tripService.getTripById(trip.getTripId());
 
-        if (existingTrip.isPresent()) return ResponseEntity.status(201).body(tripService.updateTrip(trip));
+        if (existingTrip.isPresent()) return ResponseEntity.status(201).body(new TripDTO(tripService.updateTrip(trip)));
 
         return ResponseEntity.badRequest().build();
     }
+    
+    /**
+     * HTTP POST method (/trips/{tripId}/rider)
+     * 
+     * @param tripId represents the id of the trip the person is joining
+     * @param rider represents the User joining the trip.
+     * @return The newly updated Trip object.
+     */
+    @ApiOperation(value = "Adds a user to a trip", tags = {"Rider", "Trip"})
+    @PostMapping("/{tripId}/rider")
+    public ResponseEntity<TripDTO> updateTripRider(@PathVariable("tripId") int tripId, @Valid @RequestBody UserDTO rider){
```

Comment:
If this method queries for the User object, does the request really need to include the UserDTO in the body? Alternatively, we can do a POST and DELETE to /trips/rider, and use query parameters with the tripId and userId. That cleans up the endpoints and makes sending requests easier.

Reply:
Great idea. I'll refactor and retest now